Fixed icons in KDE

RyuzakiKK opened the pull request.

RyuzakiKK committed:
Update deprecated icon names

muelli commented:
hm. https://lazka.github.io/pgi-docs/Gtk-3.0/constants.html#Gtk.STOCK_GO_BACK says

    Deprecated since version 3.10: Use the named icon “edit-cut” or the label “Cu_t”

Are you proposing a change to deprecated constants?

RyuzakiKK commented:
Ops, you are right.

I replaced the deprecated names with constants which are also deprecated.
Even if it fixed the problem with KDE, I should have used the new names for the icons.

I edited my fix (I did a rebase), this time I updated the deprecated names.
I successfully tested it on KDE, Xfce and GNOME.

muelli commented:
this has been merged a while back. Thanks a lot!
